test/fixtures/schema/variable-property.json

```diff
@@ -175,7 +175,7 @@
                                   "content": "application/json"
                                 }
                               },
-                              "content": "{\n  \"_links\": {\n    \"self\": {\n      \"href\": \"a URI\"\n    }\n  }\n}"
+                              "content": "{\n  \"_links\": {}\n}"
```

Comment:
Given the APIB, the "self" link example should be present given it is variable property with defined key?

```apib
# Group Example
# Resource [/example]

## Action [GET]

+ Response 200 (application/json)

    + Attributes
        - _links
            - *self*
                - href: a URI
```

Reply:
I think I've misinterpreted variable properties. Isn't the token `self` in `*self*` just a variable name?

Reply:
https://apiblueprint.org/documentation/mson/specification.html#322-variable-property-name

> `*rel (Custom String)* (object)`
> Where rel is a sample value for the arbitrary Property Name of a Property Member Declaration.

I believe it matches the arbitrary property name sample value section of spec.

The example itself would imply that was the intention as in the HAL spec (HAL being the content format being used) `self` is a common relation identifier.